### PR TITLE
#97 Location data import

### PR DESCRIPTION
**Pull request by kamknitt**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by koshalt** [comment]
Ok to test

**Comment by koshalt** [comment]
test this please

File: region/src/main/java/org/motechproject/nms/region/service/impl/HealthBlockImportServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    @Override
+    protected Map<String, CellProcessor> getProcessorMapping() {
+        Map<String, CellProcessor> mapping = new HashMap<>();
+        mapping.put(BID, new GetLong());
+        mapping.put(REGIONAL_NAME, new GetString());
+        mapping.put(NAME, new GetString());
+        mapping.put(HQ, new GetString());
+        final TalukaDataService talukaDataService1 = talukaDataService;
```
Comment (larubbio) [inline region/src/main/java/org/motechproject/nms/region/service/impl/HealthBlockImportServiceImpl.java]: Is there a reason this processor mapping uses a local final variable and the others don't?

**Comment by larubbio** [comment]
lgtm

**Pull request merged**